rootfs/start.sh

```diff
@@ -59,7 +59,7 @@ listen main
 EOF
 
     # Add erb code to a new template file
-    sed "/^    bind \+\*\?:/s/\*\?:\(.*\)/<%= bind_tcp('0.0.0.0', \1) %>/" \
+    sed "/^    bind \+\*\?:/s/\*\?:\(\([0-9]*\)\|\({{[^}]*}}\)\)/<%= bind_tcp('0.0.0.0', \1) %>/" \
```

Comment:
Do you mean I should break it out into a separate shell variable like the following?
```
bindport='\(\([0-9]*\)\|\({{[^}]*}}\)\)'
sed "/^    bind \+\*\?:/s/\*\?:$bindport/<%= bind_tcp('0.0.0.0', \1) %>/" \
```